src/API/Authentication.php

```diff
@@ -1,4 +1,13 @@
 <?php
```

Comment:
Summary of changes:

- Spacing and indenting (automatically with `phpcbf`)
- Docblocks
- Private properties get a `_` prefix
- Line length = 85